src/clients/SpokePoolClient.ts

```diff
@@ -168,24 +168,6 @@ export class SpokePoolClient extends BaseAbstractClient {
     return this.depositRoutes;
   }
```

Comment:
Should we also drop this, and `this.depositRoutes` entirely?

Reply:
We use this in the `scripts/disableDepositRoutes` script that's why I kept it